src/AddTodoForm.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 
-function AddTodoForm (props) {
+function AddTodoForm ({onAddTodo}) {
 
-    function handleAddTodo(event) {
-        event.preventDefault();
-        const todoTitle = event.target.title.value;
+    const [todoTitle, setTodoTitle] = React.useState('');
+
+    function handleTitleChange (e) {
+        const newTodoTitle = e.target.value;
+        setTodoTitle(newTodoTitle);
```

Comment:
Again, nothing to change here, just wanted to show you other ways of doing this.

```suggestion
        setTodoTitle(e.target.value);
```

another way (and there are pros and cons to this) is to just pass an anonymous function into the `<input>`'s `onChange` attribute like this `onChange={(e) => setTodoTitle(e.target.value)}`